README.md

```diff
@@ -117,31 +117,37 @@ $ ronin-recon worker dns/lookup
 Run the recon engine on a single domain:
 
 ```shell
-$ ronin-recon run --domain example.com
```

Comment:
If it's not too much work, could you extract the changes where you removed the old `--domain` options from the documentation into a separate PR? I could immediately merge that.

Reply:
https://github.com/ronin-rb/ronin-recon/pull/92